src/templates/formlyfield.multicheckbox.ts

```diff
@@ -23,15 +24,12 @@ import {Field} from "./field";
     `,
 })
 export class FormlyFieldMultiCheckbox extends Field {
-  constructor(private formBuilder: FormBuilder) {
-    super();
-  }
-
-  createControl(): AbstractControl {
-    let controlGroupConfig = this.templateOptions.options.reduce((previous, option) => {
-      previous[option.key] = [this.model ? this.model[option.key] : undefined];
+  static createControl(model: any, field: FormlyFieldConfig): AbstractControl {
+    let controlGroupConfig = field.templateOptions.options.reduce((previous, option) => {
+      previous[option.key] = new FormControl(model ? model[option.key] : undefined);
```

Comment:
Should we document this contract that if any template needs to  create custom controls they can provide this static method.

At present I am not sure where this can be documented :) thus this comment should not be showstopper for this PR.